bedrock/teach/templates/teach/smarton/surveillance.html

```diff
@@ -0,0 +1,368 @@
+
+        <div class="tip">
+          <h4>{{ _('Choose strong PINs and passwords.') }}</h4>
+          <p>
+            {{ _('Here are some Mozilla <a href="{url}">tips for creating a strong password for every device and account</a>.').format(url='https://support.mozilla.org/kb/create-secure-passwords-keep-your-identity-safe')|safe }}
+            {{ _('And get in the habit of changing your passwords once a year.') }}
+          </p>
+        </div>
+
+        <div class="tip">
+          <h4>{{ _('Don’t use a single password everywhere.') }}</h4>
+          <p>
+            {{ _('Would you use the same key for your front door, your car, and your safety deposit box? Probably not.') }}
+            {{ _('Keep track with a password manager instead.') }}
+            {% trans  firefox='https://support.mozilla.org/kb/password-manager-remember-delete-change-passwords',
+                      chrome='https://support.google.com/chrome/answer/95606',
+                      ie='http://windows.microsoft.com/en-us/internet-explorer/fill-in-forms-remember-passwords-autocomplete',
+                      safari='https://support.apple.com/kb/PH19249',
+                      opera='http://help.opera.com/Windows/12.00/en/password.html' %}
+              See instructions for <a href="{{ firefox }}">Firefox</a>, <a href="{{ chrome }}">Chrome</a>, <a href="{{ ie }}">Internet Explorer</a>, <a href="{{ safari }}">Safari</a> and <a href="{{ opera }}">Opera</a>.
+            {% endtrans %}
+          </p>
+        </div>
+
+        <div class="tip">
+          <h4>{{ _('Do the two-step') }}</h4>
+          <p>
+            {{ _('For the best protection, take advantage of 2-step authentication wherever it’s offered.') }}
+            <a href="http://stopthinkconnect.org/2stepsahead/resources">{{ _('Find out more.') }}</a>
+          </p>
+        </div>
+      </div>
+
+      <div class="topic with-image">
+        <div class="topic-intro">
+          <h3>{{ _('Try encryption') }}</h3>
+          <p>{{ _('Encryption is a bit like scrambling a frequency: it turns information into codes that can only be read if you have the right keys.') }}</p>
+          <p>{{ _('No software provides perfect protection from spying, but Mozilla programmers recommend these easy encryption tools to take your online security beyond basic.') }}</p>
+        </div>
+
+        {{ high_res_img('teach/smarton/surveillance/topic-encryption.png', {'width': '300', 'height': '350', 'class': 'topic-image', 'alt': ''}) }}
+
+        <div class="tip">
+          <h4>{{ _('Phone apps') }}</h4>
+          <p>{{ _('These apps use encryption to secure your phone calls, texts, picture and video communications.') }}</p>
+
+          <div class="columns columns-2">
+            <div class="col">
+              <h5>{{ _('Android') }}</h5>
+              <ul>
+                <li><a href="https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms" class="go">{{ _('TextSecure') }}</a></li>
```

Comment:
Somehow those bullets are broken on Firefox and Chrome on Android. `.main .section ul > li::before { content: '\2b25'; }`. Not a blocker because the problem exists on the current page.

![screenshot_20151221-151539](https://cloud.githubusercontent.com/assets/2929505/11940378/4b2f0726-a7f6-11e5-9ed0-83541d3a109c.png)